resources/css/filament/admin/theme.css

```
@@ -6,6 +6,118 @@
-    @apply bg-gray-100 dark:bg-gray-900;
+    @apply bg-white dark:bg-zinc-950 !border-r !border-slate-200/70 dark:!border-white/5 !shadow-xl !transition-all !duration-300;
+}
+
+.fi-sidebar-header {
+    display: none !important; /* Hide native sidebar brand */
+}
+
+/* 2. Topbar Styling */
+.fi-topbar {
+    @apply !bg-white/80 dark:!bg-zinc-950/80 !backdrop-blur-xl !border-b !border-slate-200/70 dark:!border-white/5 !sticky !top-0 !z-30 !shadow-sm;
+    height: var(--topbar-height) !important;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)